google/cloud/bigtable/row_data.py

```diff
@@ -400,7 +400,7 @@ def __init__(self, read_method, request, retry=DEFAULT_RETRY_READ_ROWS):
         self.read_method = read_method
         self.request = request
         self.retry = retry
-        self.response_iterator = read_method(request)
+        self.response_iterator = read_method(request, timeout=self.retry._deadline + 1)
```

Comment:
why is this + 1? Can you add a comment explaining?

Reply:
@kolea2 Sure, copied from the PR description.